<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>193</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QPushButton" name="button_1">
     <property name="text">
      <string>Trigger 1</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="button_2">
     <property name="text">
      <string>Trigger 2</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="button_3">
     <property name="text">
      <string>Trigger 3</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="button_4">
     <property name="text">
      <string>Trigger 4</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="button_5">
     <property name="text">
      <string>Trigger 5</string>
     </property>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>3</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
